packages/webapp/src/components/AddCropVariety/index.jsx

```diff
@@ -107,19 +191,16 @@ export default function PureAddCropVariety({
       >
         {React.cloneElement(imageUploader, {
           hookFormRegister: imageUrlRegister,
-          targetRoute: 'crop_variety',
+          targetRoute: 'crop',
```

Comment:
@MwayaB 

Just wondering if you had issues with this -- crop_variety should be correct. The folder is where all the crop_vareity photos would be stored  on Digital Ocean.

Reply:
Yes, the file upload fails when the target route is set to crop_variety

Reply:
Would you be willing to show me the error you see for that? The crop_variety route for the image has been working on beta since it was made by someone however long ago.

I personally added the crop version of the route to AddNewCrop, and renamed the attribute, when making the AddNewCrop adjustments. 

They both don't work for me locally since I am not using beta .env vars.

Reply:
Just tried it again it again to recreate the error and it looks to be working now with the target route set to 'crop_variety'

Reply:
Perfect thanks for checking that!